tests/testthat/test-read_redcap.R

```diff
@@ -370,7 +370,6 @@ test_that("read_redcap errors with bad inputs", {
   # redcap uri
   expect_error(read_redcap(123, Sys.getenv("REDCAPTIDIER_CLASSIC_API")), class = "check_character")
   expect_error(read_redcap(letters[1:3], Sys.getenv("REDCAPTIDIER_CLASSIC_API")), class = "check_character")
-  expect_error(read_redcap("https://www.google.com", Sys.getenv("REDCAPTIDIER_CLASSIC_API")), class = "cannot_post")
```

Comment:
I think this should be ok, the next test tests an unresolvable host and we test anything that _would_ give us a 405 is handled correctly here:

https://github.com/CHOP-CGTInformatics/REDCapTidieR/blob/139aee878016470cde51d47a90fa11eff6ff3922/tests/testthat/test-utils.R#L315-L316